doc/enterprise/audit_log.rst

```diff
@@ -1,40 +1,253 @@
-.. _enterprise_audit_module:
+..  _enterprise_audit_module:
 
 Audit module
 ============
 
-The audit module available in Tarantool Enterprise Edition writes messages that record Tarantool events in plain text, CSV, or JSON format.
+**Example on GitHub**: `audit_log <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/config/instances.enabled/audit_log>`_
 
-It provides detailed reports of security-related activities and helps you find and
+The audit module available in Tarantool Enterprise Edition writes messages that record Tarantool events in plain text,
+CSV, or JSON format.
+Each :ref:`event <audit-log-events-types>` is an action related to authorization and authentication, data manipulation,
+administrator activity, or system events.
+The module provides detailed reports of these activities and helps you find and
 fix breaches to protect your business. For example, you can see who created a new user
-and when:
+and when.
+
+It is up to each company to decide exactly what activities to audit and what actions to take.
+System administrators, security engineers, and people in charge of the company may want to
+audit different events for different reasons. Tarantool provides such an option for each of them.
+
+..  _audit-log-configure:
+
+Configuring audit log
+---------------------
+
+The section describes how to enable and configure audit logging and write logs to a selected destination -- a file, a pipe,
+or a system logger.
+
+Read more: :ref:`Audit log configuration reference <configuration_reference_audit>`.
+
+..  _audit-log-enable:
+
+Enable audit logging
+~~~~~~~~~~~~~~~~~~~~
+
+To enable audit logging, define the log location using the
+:ref:`audit_log.to <configuration_reference_audit_to>` option in the ``audit_log`` section of the configuration file.
+Possible log locations:
+
+*   a :ref:`file <configuration_reference_audit_file>`
+*   a :ref:`pipe <configuration_reference_audit_pipe>`
+*   a :ref:`system logger <configuration_reference_audit_syslog-server>`
+
+To disable audit logging, set the ``audit_log`` option to ``devnull``.
+
+In the configuration below, the :ref:`audit_log.to <configuration_reference_audit_to>` option is set to ``file``.
+It means that the logs are written to a file.
+In this case, you should also define a file path (for example, ``audit_tarantool.log``) using
+the :ref:`audit_log.file <configuration_reference_audit_file>` option.
+If the option is omitted, the default path is used: ``var/log/instance001/audit.log``.
```

Comment:
> the default path is used: ``var/log/instance001/audit.log``.

Please double check. Looks like its value should be dynamic: `var/log/{{ instance_name }}/audit.log`.